Add support for unit of measurement

tiagofreire-pt opened the pull request:
configuration example:

```yaml
sensor:
  - platform: ssh
    host: !secret proxmox_host
    name: 'NUC CPU Temp'
    username: !secret proxmox_user
    password: !secret proxmox_pass
    command: "sensors | grep 'Package id 0:' | cut -c17-20"
    value_template: >-
      {%- set line = value.split("\r\n") -%}
      {{ line[1] }}
    unit_of_measurement: "ºC"

tiagofreire-pt committed:
Add support for unit of measurement

LbDab reviewed (changes requested):
Add "CONF_UNIT_OF_MEASUREMENT" here in line 18:

```
from homeassistant.const import (
    CONF_NAME, CONF_HOST, CONF_USERNAME, CONF_PASSWORD,
    CONF_VALUE_TEMPLATE, CONF_COMMAND, CONF_PORT,
    STATE_UNKNOWN, CONF_UNIT_OF_MEASUREMENT)

```

tiagofreire-pt committed:
Add 'CONF_UNIT_OF_MEASUREMENT' to line 18

https://github.com/custom-components/sensor.ssh/pull/5#pullrequestreview-261944548

tiagofreire-pt commented:
@LbDab, done as requested.

Thanks!
